core/src/test/scala/org/apache/spark/SharedSparkContext.scala

```diff
@@ -29,10 +29,25 @@ trait SharedSparkContext extends BeforeAndAfterAll with BeforeAndAfterEach { sel
 
   var conf = new SparkConf(false)
 
+  /**
+   * Initialize the [[SparkContext]].  Generally, this is just called from
```

Comment:
nit: The max length of each line in the comment should also be 100, seems you are limiting that to 80?

Reply:
fixed here, not elsewhere yet